src/server/hook_func.c

```diff
@@ -3917,7 +3917,7 @@ process_hooks(struct batch_request *preq, char *hook_msg, size_t msg_len,
 			continue;
 		}
 
-		if (phook->event & HOOK_EVENT_PERIODIC) {
+		if (hook_event & HOOK_EVENT_PERIODIC) {
```

Comment:
@npadole20 can you please explain how this change is helping. As I understand, If hook_event is set to HOOK_EVENT_PERIODIC then hook_ptr points to svr_periodic_hooks. If that is the case then every phook->event should be a HOOK_EVENT_PERIODIC.